Neos.Media.Browser/Classes/ViewHelpers/FileTypeIconViewHelper.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+namespace Neos\Media\Browser\ViewHelpers;
+
+/*
+* This file is part of the Neos.Media.Browser package.
+*
+* (c) Contributors of the Neos Project - www.neos.io
+*
+* This package is Open Source Software. For the full copyright and license
+* information, please view the LICENSE file which was distributed with this
+* source code.
+*/
+
+use Neos\Media\Browser\Service\FileTypeIconService;
+use Neos\Flow\Annotations\Inject;
+use Neos\Flow\ResourceManagement\ResourceManager;
+use Neos\FluidAdaptor\Core\ViewHelper\AbstractTagBasedViewHelper;
+
+/**
+ * Renders an <img> HTML tag for a file type icon for a given filename.
+ *
+ * = Examples =
+ *
+ * <code title="Rendering a file type icon">
+ * <mediaBrowser:fileTypeIcon filename="{assetProxy.filename}" height="16" />
+ * </code>
+ * <output>
+ * (depending on the asset, no scaling applied)
+ * <img src="_Resources/Static/.../Icons/16px/jpg.png" height="16" alt="file type alt text" />
+ * </output>
+ */
+class FileTypeIconViewHelper extends AbstractTagBasedViewHelper
```

Comment:
Doesn't this exist in the Media package?

Reply:
Yes, will be consolidated.